### PR TITLE
feat(docs): Added a Spelling Error Caution Notice

### PR DESCRIPTION
**Pull request by Hello9999901**
Having helped a number of people in the Discord server, I've noticed many run into errors with writing the wrong keycode name. After discussing with Guest-42 (a person who ran into this error), he discussed the possibility of adding a caution notice as a friendly reminder to make sure the codes are spelled correctly. I am not sure if this will help a lot, so please change as you see fit :)

### CODE REVIEW

**Review by CB-42458 — APPROVED** [review]
Seems good to me, as I described in the discord server.

**Comment by caksoylar** [comment]
I think this is a pretty trivial warning for most people so I don't particularly like it. By that logic we'd have to add it to all configuration sections, because most typos in DT syntax (maybe except for label/node names etc.) will result in errors. I'll defer to other reviewers if they think it is helpful, though.

**Comment by petejohanson** [comment]
I'm a tad conflicted on this as well, TBH. Seems duplicative to add everywhere, I'm curious if it would actually help all that much as well, given users might read right past it if it's everywhere.

**Comment by Hello9999901** [comment]
> I'm a tad conflicted on this as well, TBH. Seems duplicative to add everywhere, I'm curious if it would actually help all that much as well, given users might read right past it if it's everywhere.

Hmm... I agree adding it everywhere is repetitive and will encourage skipping now that I'm thinking about it. What about only adding it on the full list, since most people would go to there for reference first?

**Comment by caksoylar** [comment]
Sorry, I seem to have missed this. I think that's a good compromise, I'd be good with that.

**Review by caksoylar — APPROVED** [review]
Thanks, looks good on the preview as well!

File: docs/src/components/codes/SpellingCaution.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
```
Comment (kurtis-lew) [inline docs/src/components/codes/SpellingCaution.jsx:2]:
```suggestion
 * Copyright (c) 2022 The ZMK Contributors
```

**Review by petejohanson — APPROVED** [review]

**Pull request merged**